packages/sampler/stories/qa-stories/VirtualList.js

```diff
@@ -34,9 +35,21 @@ const
 		);
 	};
 
-for (let i = 0; i < 1000; i++) {
-	items.push({item :'Item ' + ('00' + i).slice(-3), selected: false});
-}
+const updateDataSize = (dataSize) => {
+	const
+		itemNumberDigits = dataSize > 0 ? ((dataSize - 1) + '').length : 0,
+		headingZeros = Array(itemNumberDigits).join('0');
+
+	items.length = 0;
+
+	for (let i = 0; i < dataSize; i++) {
+		items.push({item :'Item ' + (headingZeros + i).slice(-itemNumberDigits), selected: false});
```

Comment:
It seems that we need to initialize the `StatefullSwitchItem`'s `selected` state when updating `items` array.

Reply:
`items[this.props.index].selected` and `this.state.selected` are not matched each other in `StatefulSwitchItem `.

Reply:
Oh, you're right. Hmm. any better way to clear up states of all items?

Reply:
Note: discussed with @MikyungKim and @ybsung, and concluded not to update qa-sampler. It is unclear to update all switch items when `dataSize` is changed, and qa-sampler is to test some special cases, not for general samples. I will revert changes of qa-sampler.